tests/cypress/tests/create_policy_field_validation.spec.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+/// <reference types="cypress" />
+
+describe('RHACM4K-2349 - GRC UI: [P1][Sev1][policy-grc] Create policy page: Check policy field validations', () => {
+
+  const errorMsg = 'Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:• the combined length of namespace and policy name should not exceed 63 characters• contain only lowercase alphanumeric characters, \'-\' or \'.\'• start with an alphanumeric character• end with an alphanumeric character'
+  const errorMsg2 = 'Invalid name: should only have lowercase alphanumeric characters, \'-\', or \'.\' and not begin or end with punctuation'
+//  const errorMsgRegExp = /Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:/
+  const alertMsg = 'Name already exists: pressing \'Create\' will prompt to update the existing policy'
+  const namePatterns = [
+                         ['this-is-n,ot-a-valid-name', 'default', errorMsg],  // comma inside
+                         ['-this-is-not-a-valid-name', 'default', errorMsg2], // starts with dash
```

Comment:
@ChunxiAlexLuo @dhaiducek It is interesting that in this case the error message is different. Is it on purpose?

Reply:
I don't think that's expected. The shorter one comes directly from the editor errors, which appears in a tooltip and so needs to be shorter. The full error message appears after clicking the "Create" button. If the notification wasn't closed, then maybe you'd see it flip to the shorter message, but the test looks like it clicks the "Create" button, so I don't know why this particular test would be any different. The long notification is based on a state update, so it might be possible that there's a missing state update somewhere, but I'm not sure it's worth looking in too much since there is an informative message displayed...